projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -460,16 +460,17 @@ export function startCustomerEmulation(customer, b2b = false): void {
     .should('eq', 200);
 
   cy.get('cx-customer-selection div.asm-results button').click();
-  cy.get('cx-customer-selection button[type="submit"]').click();
-
-  cy.wait(userDetailsRequestAlias).its('response.statusCode').should('eq', 200);
-  cy.get('cx-customer-emulation .cx-asm-customerInfo label.cx-asm-name').should(
-    'contain',
-    customer.fullName
-  );
-  cy.get('cx-csagent-login-form').should('not.exist');
-  cy.get('cx-customer-selection').should('not.exist');
-  cy.get('cx-customer-emulation').should('be.visible');
+  cy.get('cx-customer-selection button[type="submit"]')
```

Comment:
These changes should be reverted. (see [CXSPA-4948](https://jira.tools.sap/browse/CXSPA-4948))

Reply:
add a reminder to the PR for the epic that we either need to revert before the merge, or close CXSPA-4948 after we merged.